developer_manual/app_publishing_maintenance/release_automation.rst

```diff
@@ -0,0 +1,201 @@
+==================
+Release Automation
+==================
+
+.. sectionauthor:: Benjamin Brahmer <info@b-brahmer.de>
+
+Automation is a great thing, it prevents mistakes and makes your life easier, which gives you more time to work on all those features you wanted to implement.
+
+GitHub Actions
+--------------
+If your application's repository lives on GitHub, as many Nextcloud applications do, GitHub Actions is a great way to automate the release of your app from the git repository into the Nextcloud App Store.
+
+One easy way to get you started is to use https://github.com/R0Wi/nextcloud-appstore-push-action in your repository together with a few other actions. You can automatically build your app and publish it to the App Store. It supports pre-releases and code signing.
+To get started you create a new yaml file in the ``.github/workflows`` directory.
+
+.. code-block:: yaml
+
+    name: Build and publish app release
+
+    on:
+    release:
```

Comment:
The indentation looks off here and as it is YAML it most likely will cause issues when copied as it is. 🙈 Minde to adjust this?

Reply:
yea fixed and updated